File: 1. Fundamentos de Programación para Desarrollo Full Stack/Python/operadores.py
# Crea una función que calcule el área de un rectángulo y determine si el área es mayor a 50 usando operadores relacionales.


def area_rectangulo(base, altura):
    area = base * altura
    if area > 50:
        return f"El área del rectángulo es {area} y es mayor a 50"
    else:
        return f"El área del rectángulo es {area} y es menor a 50"


print(area_rectangulo(10, 6))
print(area_rectangulo(10, 5))
print(area_rectangulo(10, 10))

File: 1. Fundamentos de Programación para Desarrollo Full Stack/Java/operadores.java
// Escribe un programa que determine si una persona es mayor de edad usando un operador ternario.

import java.util.Scanner;

public class Operadores {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Introduce tu edad: ");
        int edad = sc.nextInt();
        sc.close();
        String resultado = (edad >= 18) ? "Eres mayor de edad" : "Eres menor de edad";
        System.out.println(resultado);
    }
}